test/images/scripts/debian-unstable.setup

```diff
@@ -76,6 +78,10 @@ curl \
 useradd -m -U -c Administrator -G sudo -s /bin/bash admin
 echo admin:foobar | chpasswd
 
+# For storaged
+echo "deb http://ftp.uk.debian.org/debian experimental main" >/etc/apt/sources.list.d/experimental.list
```

Comment:
I have just copied the url from `debian-unstable.bootstrap`, and that one is a copy of what was in the virt-builder template, I think.

We can certainly switch, in both places.